packages/components/ng-ovh-payment-method/src/legacy/payment-method-legacy.js

```diff
@@ -145,23 +145,6 @@ export default class OvhPaymentMethodLegacy {
   =            Payment Means            =
   ===================================== */
 
-  /**
-   *  Get the right v6 resource for /me/paymentMean APIs.
-   *
-   *  @param  {String} paymentMeanType The type of payment mean that will be used to determine
-   *                                   the API route (and so the right resource)
-   *  @return {ngResource}
-   */
-  getPaymentMeanResource(paymentMeanType) {
```

Comment:
Maybe we could also add a BREAKING  CHANGE description for the removal of this method

Reply:
It was a helper function used internally to get the corresponding OvhApiService endpoint for a given payment type (such as /bankAccount, /creditCard etc). It's not needed anymore since i build the url with the payment type. There is no use of this method outside of this service

Reply:
@frenautvh No use from us I agree but as this package is published it might be breaking from some apps outside manager
As package will still be released as a major it is nitpicking but it could be nice to be as accurate as possible :)

Reply:
As we didn't take time to deprecated it from the `v6.x.x` trunk with a deprecated message like [`@deprecated`](https://jsdoc.app/tags-deprecated.html) comments, maybe we can start adding a `@deprecated` message in order to be able to safely remove it from the next major release.

@frenautvh Is that make sense for you to keep this method and mark it as deprecated?

Reply:
ping @frenautvh =)

Reply:
Or the `BREAKING CHANGE:` message must also specified that this method has been removed.